backend/src/emissions/email.py

```diff
@@ -0,0 +1,72 @@
+from django.core.mail import send_mail
+from pathlib import Path
+from django.template.loader import render_to_string
+from django.utils.html import strip_tags
+import logging
+from django.core.mail import EmailMessage
+
+logger = logging.getLogger('test')
+
+class EmailClient:
+    """Handles sending emails"""
+
+    def __init__(self, template_dir: str):
+        """
+        Initialize
+        :param template_dir: Path to directory containing templates
+        """
+        self.template_dir = Path(template_dir) / 'email'
+
+    def get_template_email(self, name: str, values: str = None):
+        """
+        Get template for email text from template folder
+        :param name: Name of template
+        :param values: Values which should be replaced in template text
+        :return:
+        """
+        if not values:
+            values = {}
```

Comment:
In the paramter typings, values is a string. Here we set it as a dictionary. Is this intended?

Reply:
donce